WcaOnRails/app/views/static_pages/wca_workbook_assistant_versions.html.erb

```diff
@@ -4,7 +4,12 @@
   <h1><%= yield(:title) %></h1>
   <h2>Release Notes</h2>
   <br/>
-  <h3>v2.2 (7th April 2015)</h3>
+  <h3>v2.4 (25th September 2017)</h3>
+  <ul>
+      <li><strong>Improvement <a href="https://github.com/thewca/wca-workbook-assistant/pull/116">#116</a></strong><br>Add support for multiple attempts and groups of multiple blindfolded.
+          See more information <a href="https://github.com/thewca/worldcubeassociation.org/issues/1922">here</a>.</li>
+  </ul>
+  <h3>v2.3 (7th April 2015)</h3>
```

Comment:
Note that this was incorrectly showing up as `v2.2` in the previous version. This typo has been with us since the I pulled this content out of drupal. See c5ddd14067bee70172ccab11767313d25be51de0.